drinfomon.lic

```diff
@@ -553,41 +553,43 @@ sleep(0.1) until Char.name && !Char.name.empty?
+lirum_banks = ["Aesry Surlaenis'a", "Hara'jaal", "Mer'Kresh", "Muspar'i", "Ratha", "Riverhaven", "Rossman's Landing", "Therenborough", "Throne City"]
+dokora_banks = ["Ain Ghazal", "Boar Clan", "Chyolvea Tayeu'a", "Hibarnhvidar", "Fang Cove", "Raven's Point", "Shard"]
 
 bank_titles = {
-  "Aesry Surlaenis'a" => ['[[Tona Kertigen, Deposit Window]]'],
-  'Ain Ghazal'				=> ['Unknown Description'],
-  'Boar Clan'					=> ['Unknown Description'],
-  "Chyolvea Tayeu'a"	=> ["[[Chyolvea Tayeu'a, Teller]]"],
-  'Crossings'					=> ['[[Provincial Bank, Teller]]'],
-  'Dirge'						  => ["[[Dirge, Traveller's Bank]]"],
-  "Hara'jaal"					=> ['Unknown Description'],
-  'Hibarnhvidar'			=> ['[[Second Provincial Bank of Hibarnhvidar, Teller]]', '[[Hibarnhvidar, Teller Windows]]', '[[First Arachnid Bank, Lobby]]'],
-  'Ilaya Taipa'				=> ['Unknown Description'],
-  'Leth Deriel'				=> ['[[Imperial Depository, Domestic Branch]]'],
-  "Mer'Kresh"					=> ["[[Harti Clemois Bank, Teller's Window]]"],
-  "Muspar'i"					=> ["[[Old Lata'arna Keep, Teller Windows]]"],
-  'Ratha'						  => ['[[Lower Bank of Ratha, Cashier]]', '[[Sshoi-sson Palace, Grand Provincial Bank, Bursarium]]'],
-  "Raven's Point"				=> ["[[Bank of Raven's Point, Depository]]"],
-  'Riverhaven'				=> ['[[Bank of Riverhaven, Teller]]'],
-  "Rossman's Landing"	=> ['Unknown Description'],
-  'Shard'						  => ["[[First Bank of Ilithi, Teller's Windows]]"],
-  'Therenborough'			=> ['[[Bank of Therenborough, Teller]]'],
-  'Throne City'				=> ['[[Faldesu Exchequer, Teller]]']
+  "Ain Ghazal"         => ["Unknown Description"],
+  "Boar Clan"          => ["Unknown Description"],
+  "Crossings"          => ["[[Provincial Bank, Teller]]"],
+  "Dirge"              => ["[[Dirge, Traveller's Bank]]"],
+  "Fang Cove"          => ["[[First Council Banking, Vault]]"],
+  "Hibarnhvidar"       => ["[[Second Provincial Bank of Hibarnhvidar, Teller]]", "[[Hibarnhvidar, Teller Windows]]", "[[First Arachnid Bank, Lobby]]"],
+  "Ilaya Taipa"        => ["Unknown Description"],
+  "Leth Deriel"        => ["[[Imperial Depository, Domestic Branch]]"],
+  "Ratha"              => ["[[Lower Bank of Ratha, Cashier]]", "[[Sshoi-sson Palace, Grand Provincial Bank, Bursarium]]"],
+  "Riverhaven"         => ["[[Bank of Riverhaven, Teller]]"],
+  "Shard"              => ["[[First Bank of Ilithi, Teller's Windows]]"],
+  "Therenborough"      => ["[[Bank of Therenborough, Teller]]"],
+  "Throne City"        => ["[[Faldesu Exchequer, Teller]]"],
+  "Aesry Surlaenis'a"  => ["[[Tona Kertigen, Deposit Window]]"],
+  "Chyolvea Tayeu'a"   => ["[[Chyolvea Tayeu'a, Teller]]"],
+  "Hara'jaal"          => ["Unknown Description"],
+  "Mer'Kresh"          => ["[[Harti Clemois Bank, Teller's Window]]"],
+  "Muspar'i"           => ["[[Old Lata'arna Keep, Teller Windows]]"],
+  "Raven's Point"      => ["[[Bank of Raven's Point, Depository]]"],
+  "Rossman's Landing"  => ["Unknown Description"]
```

Comment:
* Sorted entries alphabetically
* Standardized on double-quote rather than some entries with single and some with double
* Added Fang Cove
* Replaced tabs with spaces

Reply:
> Sorted entries alphabetically

A few seem out of place to me? E.g. Aesry

Reply:
Doh -- yes, sorted _then_ realized some were using mixed single and double-quotes. I've pushed new commit to resort them now that all entries are consistently using the same quote character